docs/faq/contributing.txt

```diff
@@ -97,3 +99,9 @@ entire community, instead of prioritizing the impact on one particular user.
 This doesn't mean that we think your problem is unimportant -- just that in the
```

Comment:
I know it's not strictly related to this PR, but seeing as your fixing this file up a bit. A few lines up on line 84:

"Another reason that bugs might be ignored for while is if the bug..."

There is a missing 'a' before the word 'while'.